### DISCUSSION TITLE
Issue with process.env feature

### DISCUSSION THREAD

**Discussion by donaldkwong** [goatandsheep/react-native-dotenv#504] [Q&A]
The `process.env` feature bypasses what's specified in `allowlist` and `blocklist` and isn't affected by the `safe` option. This represents a pretty big security risk. Can this feature be disabled?

**Comment by pke** [comment]
you can use `strict` and it wont be processed

**Reply by donaldkwong** [reply]
Thanks for the response! Can you elaborate on where to apply `strict`? I don't see that in the documentation.

**Reply by pke** [reply]
Sorry I meant "safe" mode: https://github.com/goatandsheep/react-native-dotenv#safe-mode
It will not process `progress.env`

**Reply by donaldkwong** [reply]
Unfortunately, 'safe' mode will still trigger process.env variable substitution.
